dev/bots/travis_script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex
+
+export PATH="$PWD/bin:$PWD/bin/cache/dart-sdk/bin:$PATH"
+
+if [ "$SHARD" != "build_and_deploy_gallery" ]; then
+  dart ./dev/bots/test.dart
+else
+  echo "Building and deploying Flutter Gallery"
+  if [ "$TRAVIS_OS_NAME" = "linux" ]; then
+    echo "Building Flutter Gallery for Android..."
+    (cd examples/flutter_gallery; flutter doctor --android-licenses && flutter build apk --release)
+    if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then # TODO(xster): add back && [ "$TRAVIS_BRANCH" = "dev" ] after testing
+      echo "Deploying to Play Store..."
+      (cd examples/flutter_gallery/android; bundle exec fastlane deploy_play_store)
+    fi
+  elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
+    echo "Building Flutter Gallery for iOS..."
+    (cd examples/flutter_gallery; flutter build ios --release --no-codesign)
+    if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then # TODO(xster): add back && [ "$TRAVIS_BRANCH" = "dev" ] after testing
+      echo "Re-building with distribution profile and deploying to TestFlight..."
+      (cd examples/flutter_gallery/ios; bundle exec fastlane build_and_deploy_testflight)
+    fi
+  fi
+fi
+
+if [ "$TRAVIS_OS_NAME" = "linux" ] && \
+   [ "$SHARD" = "docs" ]; then
```

Comment:
let's put the if shard=docs thing in a chain with the if/else above, and then the os=linux check can go inside that if block

Reply:
Done